editor/editor_inspector.cpp

```diff
@@ -3579,7 +3579,9 @@ void EditorInspector::edit(Object *p_object) {
 
 	next_object = p_object; // Some plugins need to know the next edited object when clearing the inspector.
 	if (object) {
-		object->disconnect(CoreStringName(property_list_changed), callable_mp(this, &EditorInspector::_changed_callback));
+		if (likely(Variant(object).get_validated_object())) {
```

Comment:
You can merge the conditions (i.e. `if (object && likely(Variant(...`).

Reply:
What about the `_clear();` line?

Reply:
🤦‍♂️